engine/render/src/render/font_renderer.cpp

```diff
@@ -1129,7 +1130,7 @@ namespace dmRender
         }
 
         // Always update after flushing
-        text_context.m_TextEntriesFlushed = text_context.m_TextEntries.Size();
+        text_context.m_TextEntriesFlushed = count;
```

Comment:
The problem is/was:
We always say that we flushed all the text entries, but the m_TextEntriesFlushed is only reset when we actually do the rendering inside the FontRenderListDispatch function. When we skip a frame (or if no texts are drawn), this value will remain the same between frames, so the m_TextEntriesFlushed will still indicate that we already flushed the texts even though we technically never did.